lib/ansible/modules/cloud/amazon/ec2_placement_group_facts.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/python
+#
+# This is a free software: you can redistribute it and/or modify
```

Comment:
The license block is now much simpler: http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#copyright